.github/workflows/deploy.yml

```
@@ -0,0 +1,37 @@
+
+      - name: Setup Pages
+        uses: actions/configure-pages@v4
+
+      - name: Upload artifact
+        uses: actions/upload-pages-artifact@v3
+        with:
+          path: './public'
+
+      - name: Deploy to GitHub Pages
+        id: deployment
+        uses: actions/deploy-pages@v4
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)